MSAL/src/configuration/MSALPublicClientApplicationConfig.m

```diff
@@ -112,6 +112,7 @@ - (id)copyWithZone:(NSZone *)zone
     item->_cacheConfig = [_cacheConfig copyWithZone:zone];
     item->_verifiedRedirectUri = [_verifiedRedirectUri copyWithZone:zone];
     item->_extraQueryParameters = [_extraQueryParameters copyWithZone:zone];
+    item->_multipleCloudsSupported = _multipleCloudsSupported;
```

Comment:
Seems we didn't propagate such a property down to the broker request construction.
We might want to propagate such flag to the broker request payload, either in the extraQueryParameters or as a separate parameter:
https://github.com/AzureAD/azure-activedirectory-tokenbroker-for-objc/blob/dev/ADAuthenticationBroker/ADAuthenticationBroker/ADBrokerRequest.m#L126

Reply:
Great catch! Will fix.